### PR TITLE
[LoadFailure] Show a pretty view when loading fails and allows retrying.

### PR DESCRIPTION
**Pull request by alloy**
Fixes https://github.com/artsy/eigen/issues/1429

loadfailureview

### CODE REVIEW

File: lib/app_registry.js [hunk 1]
```diff
@@ -2,45 +2,55 @@
...
+    super(props)
+    this.state = { retrying: false }
+  }
 
+  render() {
     // https://facebook.github.io/relay/docs/guides-root-container.html
-    return (<RootContainer
-             Component={Containers.Artist}
-             route={artistRoute}
+    return (<Relay.RootContainer
```
Comment (orta) [inline lib/app_registry.js]: I assume you'd want this on the Home too? Perhaps the re-try work could go into it's own artsy-relay-root container?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/8c7177721d3b870580bbbe10d4e7235e9a4bf09d/lib/app_registry.js">lib/app_registry.js</a> &amp; <a href="https://github.com/artsy/emission/blob/8c7177721d3b870580bbbe10d4e7235e9a4bf09d/lib/components/load_failure_view.js">lib/components/load_failure_view.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: lib/app_registry.js (hunk 1 above)
Comment (alloy) [inline lib/app_registry.js]: That's exactly what's done here ;)

**Comment by sarahscott** [comment]
🔥

File: lib/app_registry.js (hunk 1 above)
Comment (orta) [inline lib/app_registry.js]: hahahasjlabflasbfjsdbglsg

oh yeah

Comment (orta) [inline lib/app_registry.js]: I'm a pro React Nativer with 15 years experience

Comment (orta) [inline lib/app_registry.js]: I was just testing you

**Pull request merged**